src/main/java/de/btegermany/terraplusminus/gen/tree/TreePopulator.java

```diff
@@ -126,7 +127,7 @@ public void populate(@NotNull WorldInfo worldInfo, @NotNull Random random, int x
                                 waterY = data.waterHeight(valueX, valueZ);
                                 state = data.surfaceBlock(valueX, valueZ);
                             } catch (IndexOutOfBoundsException e) {
-                                e.printStackTrace();
+                                Terraplusminus.instance.getLogger().warning("Chunk boundary overflow when attempting to generate a tree at chunk " + x + "/" + z);
```

Comment:
I would suggest using parameterized logging.

```suggestion
                                Terraplusminus.instance.getLogger().warning("Chunk boundary overflow when attempting to generate a tree at chunk {}/{}", x, z);
```

Reply:
That's not supported by JUL loggers. Considering your other comments, I feel like a full migration to SL4J is warranted and I will be doing that.

Reply:
Done